core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -766,7 +766,7 @@ class BrokerServer(
         CoreUtils.swallow(dataPlaneRequestHandlerPool.shutdown(), this)
       if (dataPlaneRequestProcessor != null)
         CoreUtils.swallow(dataPlaneRequestProcessor.close(), this)
-      authorizer.foreach(Utils.closeQuietly(_, "authorizer"))
+      authorizerPlugin.foreach(Utils.closeQuietly(_, "authorizer"))
```

Comment:
Should we rename "authorizer" to "authorizer plugin" for clarity? Since we explicitly close the authorizer in `Plugin#close()`.